src/firefly/js/templates/lightcurve/LcUtil.jsx

```diff
@@ -146,10 +146,10 @@ export function getInitialDefaultValues(labelWidth, missionName) {
             return commonDefault;
         case 'wise':
             const wiseDefault = {
-                [LC.META_FLUX_BAND]: Object.assign(getTypeData(LC.META_FLUX_BAND, '',         '' +
-                    'Select WISE band for images to be displayed',         'Image display:', 70)),
-                [LC.META_ERR_CNAME]: Object.assign(getTypeData(LC.META_ERR_CNAME, '', 
-                    'value error column name',        'Error Column:', labelWidth))
+                [LC.META_FLUX_BAND]: Object.assign(getTypeData(LC.META_FLUX_BAND, '',        '' +
+                    'Select WISE band for images to be displayed',        'Image display:', 70)),
+                [LC.META_ERR_CNAME]: Object.assign(getTypeData(LC.META_ERR_CNAME, '',
+                    'value error column name',       'Error Column:', labelWidth))
```

Comment:
FYI: I removed a few non-ascii characters from here.  However, the code could use some cleaning up.  Object.assign() is used with only 1 parameter, why?  Also, why is there a `'' + 'Select...'`